.github/workflows/issue-manager.yml

```
@@ -39,5 +39,9 @@ jobs:
               "waiting": {
                 "delay": 2628000,
                 "message": "As this PR has been waiting for the original user for a while but seems to be inactive, it's now going to be closed. But if there's anyone interested, feel free to create a new PR."
-              }
+              },
+              "invalid": {
+                "delay": 0,
+                "message": "This was marked as invalid and will be closed now. If this is an error, please provide additional details."
+              },
```

```suggestion
              }
```

Let's remove this last comma, otherwise it's not valid JSON and it would break on that. :nerd_face:

---

I hadn't noticed, my bad :sweat_smile: